about.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Taas Real Estate</title>
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/semantic-ui@2.4.2/dist/semantic.min.css">
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.2/font/bootstrap-icons.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link rel="stylesheet" href="css/about.css">
+</head>
+<body>
+ <header>
+    <nav class="topNav">
+        <a href="#"><i class="bi bi-facebook"></i></a>
+        <a href="#"><i class="bi bi-twitter"></i></a>
+        <a href="#">English</a>
+        <a href="#">English</a>
+        <a href="#" class="signup">Logout</a>
+    </nav>
+    <!--Navigation mobile-->
+        <nav class="menu_mobile">
+                <div class="curtain">
+                    <a href="#" class="close">&times;</a>
+                    <div class="curtain-content">
+                        <a class="menu-link" href="#home">Properties</a>
+                        <a class="menu-link" href="#about">Blog</a>
+                        <a class="menu-link" href="#contact">Report</a>
+                        <a class="menu-link" href="#contact">Contact Us</a>
+                    </div>
+                </div>
+
+                <i class="fa fa-bars open"></i>
+        </nav>
+    <!--Navigation desktop-->
+        <nav class="menu_desktop">
+          <div class="logo">
+            <img src="images/logo.png" alt="Logo-Image">
+            <a href="index.html">Taas Homes</a>
+          </div>
+            <div class="nav_menu">
+                <a class="menu-link" href="#home">Properties</a>
+                <a class="menu-link" href="#about">Blog</a>
+                <a class="menu-link" href="#contact">Report</a>
+                <a class="menu-link" href="#contact">Contact Us</a>
+            </div>
+        </nav>
+</header>
+<!--Home section-->
+<section class="introduction">
+<a href="index.html" ><i class="fa fa-home" aria-hidden="true"></i>&nbsp;&nbsp;Home</a>
+  <div class="content ">
+  <h1>10 Units Left! Ready to move in 2&3 Beds</h1>
```

Comment:
kindly change the h1 text color here in the about page to match the given design 😄 
![image](https://user-images.githubusercontent.com/87448628/181752294-4366cca4-e3a3-49c6-9a74-1dbb73834dc7.png)